transformer_engine/common/fused_attn/fused_attn_f16_arbitrary_seqlen.cu

```diff
@@ -208,6 +220,10 @@ void fused_attn_arbitrary_seqlen_fwd_impl(
                          .set_causal_mask_bottom_right(is_bottom_right)
                          .set_attn_scale(attn_scale);
 
+      if (cudnn_runtime_version >= 90200 && window_size_left != s_q) {
+        sdpa_options.set_sliding_window_length(window_size_left);
```

Comment:
I didn't find out where `window_size_right` is passed to cuDNN. Will cuDNN support that? If not, what is the behavior when user passed different `window_size_right` and `window_size_left`, e.g. (2, 3).

Reply:
cuDNN attention only supports (>=0, 0) which is with causal mask implied. But I've requested to them that we want both left and right half of the window. I added `window_size_right` in the API to be future-proof, but right now any `window_size_right != 0` would disable the cuDNN path.

Reply:
Make sense, but would be great to add NVTE_CHECK here before cuDNN support it.

Reply:
We do have `nvte_get_fused_attn_backend` [here](https://github.com/cyanguwa/TransformerEngine/blob/28c883e1d05557d407fd4a9bb0a23537b0a60d62/transformer_engine/common/fused_attn/fused_attn.cpp#L175-L176) which is called by every `nvte_fused_attn_fwd/bwd_xxx` API. I think it's sufficient.